docs/docs/05-migrations/12-v8-to-v9.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I added this doc to drop a note of API breaking changes that this PR might include. I will also add later on some other notes for other API-breaking changes that have already been merged to main.

Reply:
Can unbreak the event key changes in the backport! :)